src/util/constants.tsx

```diff
@@ -0,0 +1,25 @@
+const bydels = [
+  'Alna',
+  'Bjerke',
+  'Frogner',
+  'Gamle Oslo',
+  'Grorud',
+  'Grünerløkka',
+  'Nordre Aker',
+  'Nordstrand',
+  'Sagene',
+  'St. Hanshaugen',
+  'Stovner',
+  'Søndre Nordstrand',
+  'Ullern',
+  'Vestre Aker',
+  'Østensjø',
+];
+
+// To fetch a question in a given language, call questions.<ISO 2 digit language code>.<form value>
```

Comment:
internationalization is gonna be a whole project. if you're curious, start digging into react-i18n which is the framework we're gonna use